core/src/main/resources/lib/layout/main-panel.jelly

```diff
@@ -30,7 +30,11 @@ THE SOFTWARE.
 
   <j:if test="${mode=='main-panel'}">
     <j:if test="${app.isQuietingDown()}">
-      <div id='shutdown-msg'>${%Jenkins is going to shut down}</div>
+      <div id='shutdown-msg'>${%Jenkins is going to shut down}
```

Comment:
Would it make sense to use the opportunity of these parts being worked on to think about the wording?

There are other reasons than a planned restart to quiet down the instance, and since we're now adding an explanation what's going on, the built-in messaging could more generically just state "Jenkins is quieting down" or "Jenkins is suspending execution of further tasks" or something like that.

Thoughts?

Reply:
I do not mind and can do it. But a little sorry to lose translations

Reply:
Imho can be done in a follow-up, we can create a ticket for it